app/views/layouts/_head_tag_content.html.erb

```diff
@@ -25,8 +25,13 @@ signed in %>
 
 <%= render 'shared/appearance_styles' %>
 
-<!-- Google Analytics -->
-<%= render partial: '/ga', formats: [:html] %>
+<% unless Hyrax.config.analytics == 'false' %>
```

Comment:
i think this `unless` isn't functional. `Hyrax.config.analytics` is casting to a boolean, so it should never `== 'false'`? am i missing anything here?